app/routes/events/components/JoinEventForm.tsx

```diff
@@ -462,60 +459,62 @@ const JoinEventForm = ({ title, event, registration }: Props) => {
                           />
                         )}
 
-                        <div>
-                          <label
-                            htmlFor={feedbackName}
-                            className={styles.feedbackLabel}
-                          >
-                            {feedbackLabel}
-                          </label>
-                          <Flex alignItems="center" gap="var(--spacing-md)">
-                            {spyValues((values: FormValues) => (
-                              <Field
-                                id={feedbackName}
-                                placeholder="Kommentar"
-                                name={feedbackName}
-                                component={TextInput.Field}
-                                className={styles.feedbackText}
-                                onKeyDown={(e) => {
-                                  if (e.key === 'Enter' && registration) {
-                                    // Prevent user from unregistering by pressing enter
-                                    e.preventDefault();
-
-                                    if (!pristine) {
+                        {event.feedbackRequired && (
```

Comment:
Correct me if I'm wrong - haven't run this locally yet - but now the feedback is only displayed if the question is mandatory right?

In the aministrate/edit page there is a possibility to check yes to a question, and write the question, without it being mandatory.. would it not then be better for this to be shown if the feedbackDescription is set OR the feedback is required? (or just if feedbackDescription is set - as that should also be the case if it is required)

<img width="343" alt="image" src="https://github.com/user-attachments/assets/494715eb-0140-4bb5-862b-2a068b542b44">

Or change this thingy as well to only support a mandatory question

Reply:
Yes you're totally right! Nice catch 😍 I updated the code now so that the comment field will be shown if a feedback description has been set